lister/list.go

```diff
@@ -66,6 +57,16 @@ func (l *RealLister) List(opts ListOptions) (model.SeshSessions, error) {
 		fullOrderedIndex = fullOrderedIndex[:destIndex]
 	}
 
+	if opts.HideAttached {
+		attachedSession, _ := GetAttachedTmuxSession(l)
+		for i, index := range fullOrderedIndex {
+			if fullDirectory[index].Name == attachedSession.Name {
+				fullOrderedIndex = slices.Delete(fullOrderedIndex, i, i+1)
+				break
```

Comment:
1. Does it seem right to break here? Is it possible to have two sessions with the same `.Name`?
2. If it's not possible to have two sessions with the same `.Name`, isn't the active session always first in the list? So Couldn't I skip the iteration and just check `fullDirectory[fullOrderedIndex[0]].Name` against `attachedSession.Name` to save a bunch of CPU cycles?